### PR TITLE
Implement question 5 (Can you offer space?)

### PR DESCRIPTION
**Pull request by brucebolt**
Adds an additional question and all the relevant logic to move to the next/previous questions.

Also introduces a placeholder route for the next question.

Screenshot 2020-03-21 at 09 57 09

### CODE REVIEW

File: app/controllers/coronavirus_form/offer_space_controller.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      radio: offer_space,
+    )
+
+    if invalid_fields.any?
+      flash.now[:validation] = invalid_fields
+      render "coronavirus_form/#{PAGE}"
+    elsif session["check_answers_seen"]
+      redirect_to controller: "coronavirus_form/check_answers", action: "show"
+    else
+      redirect_to controller: "coronavirus_form/offer_advice", action: "show"
```
Comment (bilbof) [inline app/controllers/coronavirus_form/offer_space_controller.rb:26]: This will need to be changed to the 'what kind of space' controller

**Review by richardTowers — APPROVED** [review]

**Review by bilbof — APPROVED** [review]
Looks good, section 5.1 has been added since but that could come in another PR

File: app/controllers/coronavirus_form/offer_space_controller.rb (hunk 1 above)
Comment (issyl0) [reply app/controllers/coronavirus_form/offer_space_controller.rb:26]: Ah, you chose "offer advice". I went with "expert advice"?

**Comment by brucebolt** [comment]
Great.  I'll write a card to implement 5.1.

File: app/controllers/coronavirus_form/offer_space_controller.rb (hunk 1 above)
Comment (brucebolt) [reply app/controllers/coronavirus_form/offer_space_controller.rb:26]: @issyl0 feel free to change this in your PR.  Mine was just a placeholder so I could add a test for the redirect.

**Pull request merged**